docs/conf.py

```diff
@@ -307,5 +308,5 @@
 
 # Wrap text in tables
 # https://github.com/snide/sphinx_rtd_theme/issues/117#issuecomment-41571653
-def setup(app):
-    app.add_stylesheet("theme_overrides.css")
+# def setup(app):
+#     app.add_stylesheet()
```

Comment:
Could you elaborate? Is the comment above no longer relevant? Should we remove these 4 lines? I’m worried local and ReadTheDocs renderings may differ, and this may break the latter.